### DISCUSSION TITLE
How to delete the content of a field in an API ?

### DISCUSSION THREAD

**Discussion by Barbapapazes** [adonisjs/core#2691] [Help] [answered]
Hello,

I have a user validator with lastName and firstName. Something like this 
```js
lastName: schema.string.optional({ trim: true }, [rules.maxLength(30)]),
firstName: schema.string.optional({ trim: true }, [rules.maxLength(30)]),
```

In my frontend, I have a form which send the input data. If the user wants to delete his lastName, he removes data from the filed so this is the send data
```json
{
  "lastName": "",
  "firstName": "A great name"
}
``` 

But the fact is the output from the validator is juste 
```json
{
  "firstName": "A great name"
}
```

So, the user is unable to remove the content from a field. Is there a way to do-it? Have I miss something ?

**Comment by Barbapapazes** [comment]
See's to be related to #1388, will try it

**Comment by thetutlage** [comment, marked as answer]
You can perform the update as follows:

```ts
const user = await User.find(theUserId)

user.firstName = payload.firstName || null
user. lastName = payload. lastName || null

await user.save()
```

**Reply by Barbapapazes** [reply]
Yep, so I follow #1388 and your advice and it works well.

It seems that `existsStrict` is not well documented ! But thanks for your fast answer ! 🤗
